src/tests/unit/tests/reports/package/__snapshots__/integration.test.ts.snap

```diff
@@ -1,5 +1,5 @@
tanceDetailsCard\\" tabindex=\\"0\\" aria-selected=\\"false\\" aria-label=\\"html card\\" role=\\"row\\"><table class=\\"reportInstanceTable\\"><tbody><tr class=\\"row\\"><th class=\\"label\\">Path</th><td class=\\"instanceListRowContent\\">html</td></tr><tr class=\\"row\\"><th class=\\"label\\">Snippet</th><td class=\\"instanceListRowContent contentSnipppet\\">&lt;html class=&quot;deque-axe-is-ready&quot;&gt;</td></tr><tr class=\\"row\\"><th class=\\"label\\">How to fix</th><td class=\\"instanceListRowContent\\"><div class=\\"howToFixContent\\"><div><div class=\\"insights-fix-instruction-title\\">Fix the following:</div><ul class=\\"insights-fix-instruction-list\\"><li>Document does not have a main landmark</li></ul></div></div></td></tr></tbody></table></div></div></li></ul></div></div><div class=\\"collapsible-container collapsibleRuleDetailsGroup collapsed\\"><div class=\\"title-container\\" role=\\"heading\\" aria-level=\\"3\\"><button class=\\"collapsible-control\\" aria-expanded=\\"false\\" aria-controls=\\"content-container-page-has-heading-one\\" aria-label=\\"page-has-heading-one 1 Failed Ensure that the page, or at least one of its frames contains a level-one heading\\"><span class=\\"rule-detail\\"><span class=\\"outcomeChipContainer\\"><span aria-hidden=\\"true\\"><span class=\\"outcome-chip outcome-chip-fail\\" title=\\"1 Failed\\"><span class=\\"icon\\"><span class=\\"check-container\\"><svg width=\\"16\\" height=\\"16\\" viewBox=\\"0 0 16 16\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><circle cx=\\"8\\" cy=\\"8\\" r=\\"8\\" fill=\\"#E81123\\"></circle><path d=\\"M10.9837 6.27639C11.3352 5.92491 11.3352 5.35507 10.9837 5.00359C10.6322 4.65212 10.0624 4.65212 9.7109 5.00359L7.99722 6.71728L6.28375 5.00381C5.93227 4.65234 5.36242 4.65234 5.01095 5.00381C4.65948 5.35528 4.65948 5.92513 5.01095 6.2766L6.72443 7.99007L4.9837 9.7308C4.63222 10.0823 4.63222 10.6521 4.9837 11.0036C5.33517 11.3551 5.90502 11.3551 6.25649 11.0036L7.99722 9.26287L9.73816 11.0038C10.0896 11.3553 10.6595 11.3553 11.011 11.0038C11.3624 10.6523 11.3624 10.0825 11.011 9.73101L9.27001 7.99007L10.9837 6.27639Z\\" fill=\\"white\\"></path></svg></span></span><span class=\\"count\\" aria-hidden=\\"true\\"> 1</span><span class=\\"screen-reader-only\\">1 Failed</span></span></span></span><span><span class=\\"rule-details-id\\">page-has-heading-one</span>: <span class=\\"rule-details-description\\">Ensure that the page, or at least one of its frames contains a level-one heading</span></span></span></button></div><div id=\\"content-container-page-has-heading-one\\" class=\\"collapsible-content\\" aria-hidden=\\"true\\"><div class=\\"ruleMoreResources\\"><div class=\\"moreResourcesTitle\\">Resources for this rule</div><span class=\\"ruleDetailsId\\"><a target=\\"_blank\\" href=\\"https://dequeuniversity.com/rules/axe/3.3/page-has-heading-one?application=webdriverjs\\" class=\\"ms-Link insights-link root-49\\">More information about page-has-heading-one</a></sp
```

Comment:
is there a way to pretty print the html so that it is easier to find out what changed?

Reply:
I was wondering the same thing.